en/intro-concepts.md

```diff
@@ -304,6 +304,10 @@ Deploy.
 Allocates (reserves) a node to the MAAS user performing the action (and
 currently logged in). Changes a node's status from 'Ready' to 'Allocated'.
 
+With the CLI, it is necessary to perform this action prior to deploying. With
+the web UI it is done automatically for the user. Acquiring in the web UI is
+used for machine reservation.
```

Comment:
It's only that when we have to write about it, the inconsistancy sticks out to the reader, who may then blame the documentation. But I agree, writing our way around bad terminology isn't the best solution.